sunpy/map/mapsequence.py

```diff
@@ -271,6 +271,12 @@ def plot(self, axes=None, resample=None, annotate=True,
         plot_function : function
             A function to be called as each map is plotted.
             For more information see `sunpy.visualization.animator.MapSequenceAnimator`.
+        norm : astropy.visualization.mpl_normalize.ImageNormalize 
```

Comment:
```suggestion
        norm : `matplotlib.colors.Normalize` or `str`
```

As https://matplotlib.org/stable/api/cm_api.html#matplotlib.cm.ScalarMappable.set_norm suggests.

I have no idea how a string would work for this one.

Reply:
You don't need to apologize, you are not on a schedule. 

Changes look good to me.